Misc/NEWS.d/next/Library/2025-09-07-12-43-51.gh-issue-138614.7-4rrs.rst

```diff
@@ -0,0 +1,2 @@
+:mod:`site`: correctly synchronize the ``purelib`` installation scheme with
+the one defined in :mod:`sysconfig`.
```

Comment:
After some thoughts, what I wrote is a bit too obscure. Let's mention the public API:

```suggestion
:mod:`site`: correctly synchronize the ``purelib`` installation scheme with
the one defined in :mod:`sysconfig`. This affects the default user-specific
site-packages directory path (see :func:`site.getusersitepackages`).
```